src/router/index.js

```diff
@@ -12,9 +12,9 @@ const routes = [
     component: Home,
   },
   {
-    path: '/detail/:id',
-    name: 'Detail',
-    component: () => import(/* webpackChunkName: "detail" */ '../views/details/details.vue'),
+    path: '/detail/:id?',
+    name: 'detail',
```

Comment:
please use detail or details. I recommend calling it details

Reply:
done

Reply:
done